.github/ISSUE_TEMPLATE/bug_report_template.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''

```

Comment:
add corresponding labels

Reply:
@vishakha-lall can you check the paths of these files? Do we need to put it that way?